packages/mcp-server/src/server/store/palette-store.test.ts

```
@@ -16,22 +29,57 @@ const {
+interface StoreFixture {
+  // Exposed only so a future failure can sniff the underlying connection;
+  // tests do not need it directly.
+  db: Awaited<ReturnType<typeof createIsolatedDb>>['db']
+}
 
-  afterEach(async () => {
-    await rm(tempDir, { recursive: true, force: true })
-  })
+const test = baseTest.extend<{ store: StoreFixture }>({
+  store: [
+    async ({}, use) => {
```

Biome (2.4.13)

[error] Unexpected empty object pattern.

(lint/correctness/noEmptyPattern)